pontoon/teams/urls.py

```diff
@@ -1,78 +1,68 @@
+    path(
+        "teams/<locale:locale>/",
         RedirectView.as_view(url="/%(locale)s/", permanent=True),
     ),
-    # Team contributors
-    url(
-        r"^(?P<locale>[A-Za-z0-9\-\@\.]+)/contributors/$",
-        views.team,
-        name="pontoon.teams.contributors",
-    ),
-    # Team bugs
-    url(
-        r"^(?P<locale>[A-Za-z0-9\-\@\.]+)/bugs/$", views.team, name="pontoon.teams.bugs"
-    ),
-    # Team info
-    url(
-        r"^(?P<locale>[A-Za-z0-9\-\@\.]+)/info/$", views.team, name="pontoon.teams.info"
-    ),
-    # Team permissions
-    url(
-        r"^(?P<locale>[A-Za-z0-9\-\@\.]+)/permissions/$",
-        views.team,
-        name="pontoon.teams.permissions",
-    ),
-    # Legacy url for permissions management
-    url(
-        r"^(?P<locale>[A-Za-z0-9\-\@\.]+)/manage/$",
-        RedirectView.as_view(url="/%(locale)s/permissions/", permanent=True),
-        name="pontoon.teams.manage",
-    ),
-    # AJAX view: Team projects
-    url(
-        r"^(?P<locale>[A-Za-z0-9\-\@\.]+)/ajax/$",
-        views.ajax_projects,
-        name="pontoon.teams.ajax.projects",
-    ),
-    # AJAX view: Team contributors
-    url(
-        r"^(?P<code>[A-Za-z0-9\-\@\.]+)/ajax/contributors/$",
-        views.LocaleContributorsView.as_view(),
-        name="pontoon.teams.ajax.contributors",
-    ),
-    # AJAX view: Team info
-    url(
-        r"^(?P<locale>[A-Za-z0-9\-\@\.]+)/ajax/info/$",
-        views.ajax_info,
-        name="pontoon.teams.ajax.info",
-    ),
-    # AJAX view: Update team info
-    url(
-        r"^(?P<locale>[A-Za-z0-9\-\@\.]+)/ajax/update-info/$",
-        views.ajax_update_info,
-        name="pontoon.teams.ajax.update-info",
-    ),
-    # AJAX view: Team permissions
-    url(
-        r"^(?P<locale>[A-Za-z0-9\-\@\.]+)/ajax/permissions/$",
-        views.ajax_permissions,
-        name="pontoon.teams.ajax.permissions",
-    ),
-    # AJAX: Request projects to be added to locale
-    url(
-        r"^(?P<locale>[A-Za-z0-9\-\@\.]+)/request/$",
-        views.request_item,
-        name="pontoon.teams.request.projects",
+    path(
+        "<locale:locale>/",
+        include(
+            [
+                # Team contributors
+                path("contributors/", views.team, name="pontoon.teams.contributors",),
+                # Team bugs
+                path("bugs/", views.team, name="pontoon.teams.bugs",),
+                # Team info
+                path("info/", views.team, name="pontoon.teams.info",),
+                # Team permissions
+                path("permissions/", views.team, name="pontoon.teams.permissions",),
+                # Legacy url for permissions management
+                path(
+                    "manage/",
+                    RedirectView.as_view(
+                        url="/%(locale)s/permissions/", permanent=True
+                    ),
+                    name="pontoon.teams.manage",
+                ),
+                # AJAX view: Team projects
```

Comment:
Should we add another nesting level for the AJAX views, like in the projects module?

Reply:
Done